### PR TITLE
Adjustments to feature preview landing pages

### PR DESCRIPTION
**Pull request by rezrah**
## Summary

Resolves QA feedback for feature preview landing pages, and adds a new option to `Box`.

Closes https://github.com/github/primer/issues/2656 https://github.com/github/primer/issues/2658 https://github.com/github/primer/issues/2657 https://github.com/github/primer/issues/2653

🔗 [Storybook](https://primer-41a80cc014-26139705.drafts.github.io/storybook/?path=/story/recipes-feature-previews-level-2--level-two-playground)

## List of notable changes:

- Resolves all QA feedback in above issues
- Added `0` option to Box, which is required to 'reset' paddings defined exclusively for lower viewports
- Fixes a border color discrepancy in `FormControl`

## What should reviewers focus on?

- Verify feedback is resolved

## Steps to test:

1. Review the stories

## Contributor checklist:

- [ ] All new and existing CI checks pass
- [x] Tests prove that the feature works and covers both happy and unhappy paths
- [x] Any drop in coverage, breaking changes or regressions have been documented above
- [x] New visual snapshots have been generated / updated for any UI changes
- [x] All developer debugging and non-functional logging has been removed
- [x] Related issues have been referenced in the PR description

## Reviewer checklist:

- [ ] Check that pull request and proposed changes adhere to our [contribution guidelines](../../CONTRIBUTING.md) and [code of conduct](../../CODE_OF_CONDUCT.md)
- [ ] Check that tests prove the feature works and covers both happy and unhappy paths
- [ ] Check that there aren't other open Pull Requests for the same update/change

### CODE REVIEW

**Comment by jesussandreas** [comment]
Hey @rezrah,

Thank you for getting those changes in so quickly!

I just have a couple of small changes and that's all.

---

1. The FAQ spacing between headline and questions now has 128px instead of 80px.

Screenshot 2023-09-18 at 11 05 06 AM

 2. Can we increase the spacing on the river components from `8px` to `16px`?
 
Screenshot 2023-09-18 at 11 16 30 AM

3. Can we increase the bottom spacing on the `side by side forms` to `80px from 24px`?
Screenshot 2023-09-18 at 11 19 37 AM

That's all. 

APPRECIATE YOU

**Comment by jesussandreas** [comment]
One more thing, I don't want to be annoying but seems we are still getting some issues on the responsiveness of the forms, hence why the 50 50 layout of buttons. i think will help address those. but also seems there's some extra padding on the right?

Screenshot 2023-09-18 at 11 24 03 AM

**Comment by rezrah** [comment]
> One more thing, I don't want to be annoying but seems we are still getting some issues on the responsiveness of the forms, hence why the 50 50 layout of buttons. i think will help address those. but also seems there's some extra padding on the right?
> 
> Screenshot 2023-09-18 at 11 24 03 AM

@jesussandreas The 50:50 layout for those buttons doesn't work great in all sizes, because the left button will always require more space. 👇 
Screenshot 2023-09-19 at 10 47 47

How about we stack them until the required space becomes available? 👇 

https://github.com/primer/brand/assets/13340707/5ff8cc7d-2771-4fa3-9e99-2a2532478276

**Comment by jesussandreas** [comment]
> > One more thing, I don't want to be annoying but seems we are still getting some issues on the responsiveness of the forms, hence why the 50 50 layout of buttons. i think will help address those. but also seems there's some extra padding on the right?
> 
> @jesussandreas The 50:50 layout for those buttons doesn't work great in all sizes, because the left button will always require more space. 👇 Screenshot 2023-09-19 at 10 47 47
> 
> How about we stack them until the required space becomes available? 👇
> 
>  Screen.Recording.2023-09-19.at.10.48.43.mov 
> All other feedback in previous comment also resolved @jesussandreas. cc. @natalie-iv Let me know if you have any feedback too 🙏

this works great, thank you!

**Review by jesussandreas — APPROVED** [review]
looks good to go!

**Comment by natalie-iv** [comment]
great solution staking the buttons, works for me as well

File: .changeset/tough-planets-reply.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+'@primer/react-brand': patch
+---
+
+Add `0` option to Box spacing props to reset defined paddings at larger viewports.
```
Comment (josepmartins) [inline .changeset/tough-planets-reply.md:5]: What about using `none` instead? 

I think we use that to reset defined spaces in Primer Web components: https://primer.style/design/components/page-layout/react#pagelayout

**Review by josepmartins — APPROVED** [review]
Looks good to me!

Left a non-blocker comment 👇

**Review by natalie-iv — APPROVED** [review]

File: .changeset/tough-planets-reply.md (hunk 1 above)
Comment (rezrah) [reply .changeset/tough-planets-reply.md:5]: That's a good shout 👍. We have precedent for this in PB already, as Hero and a few other components already have 'none' options. Not sure how I missed that 😅

**Pull request merged**